podcastfy/text_to_speech.py

```diff
@@ -38,8 +40,10 @@ def __init__(self, model: str = 'openai', api_key: Optional[str] = None):
 		elif self.model == 'openai':
```

Comment:
Just a minor comment, in the future we should have all TSS classes implementing a common ABC to avoid those if, elif, else constructions. But for now LGTM! @brumar is working on a PR to implement this intended level of abstraction.